hedy.py

```diff
@@ -1854,7 +1856,10 @@ def create_grammar(level, lang="en"):
     # ready? Save to file to ease debugging
     # this could also be done on each merge for performance reasons
     save_total_grammar_file(level, result, lang)
-
+    global lang_commands
+    global en_commands
```

Comment:
global object are fine, but we then we don't want to fill them again for each user (when the function is called). If we create them once and make sure they are cached this will reduce the memory load a lot. You can take a look at some other parts of the code, but we can for example create one global object like this at the start of the file:

```
COMMANDS = []
for language in keyword_languages.keys():
    COMMANDS[language] = .... #get commands for this language
```

Than we can simply call this list of language commands everywhere without the need of creating it again!

Reply:
This has become redundant as I just implemented something very similar in #2493 that you can use for this problem.

Reply:
How do I access what's inside a "website.yaml_file.YamlFile object"?

Reply:
Are you referring to a datatype? If it's a reference to a file you can read the content by using something like `.to_dict()`. But I'm not sure if you need this as the `COMMANDS` dictionary already contains the commands for each language as a dictionary.